_tutorials/install-add-fdroid-repo.html

```diff
@@ -6,20 +6,20 @@
 ---
 <section>
 
-    <p><span class="text-info">Team NewPipe's F-Droid repository allows for updates to NewPipe to be available faster than on the default F-Droid repository.</span>
+    <p><span class="text-info">Team NewPipe's F-Droid repository allows for updates to NewPipe to be available faster than on the default F-Droid repository. This is the recommended way of installing NewPipe.</span>
```

Comment:
I don't recall any discussion with the team about the preferred installation method. Why don't we do that first?

Reply:
Since we repeatedly guide users to this document, I thought that was implicitly the case.

Reply:
Voting done, I think?